groups/cdn/cloudfront.tf

```diff
@@ -0,0 +1,85 @@
+resource "aws_cloudfront_distribution" "cdn" {
+  enabled             = true
+  http_version        = "http2and3"
+  is_ipv6_enabled     = true
+  default_root_object = "index.html"
+
+  origin {
+    domain_name              = aws_s3_bucket.s3_bucket.bucket_regional_domain_name
+    origin_id                = local.cloudfront_s3_origin_id
+    origin_access_control_id = aws_cloudfront_origin_access_control.cdn.id
+  }
+
+  default_cache_behavior {
+    allowed_methods  = ["GET", "HEAD"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = local.cloudfront_s3_origin_id
+
+    viewer_protocol_policy = "redirect-to-https"
+    min_ttl                = var.min_ttl
+    default_ttl            = var.default_ttl
+    max_ttl                = var.max_ttl
+    compress               = true
+  }
+
+  restrictions {
+    geo_restriction {
+      restriction_type = "none"
+    }
+  }
+
+  viewer_certificate {
+    cloudfront_default_certificate = true
+  }
+}
+
+cache_policy_id          = aws_cloudfront_cache_policy.cache_policy.id
+origin_request_policy_id = aws_cloudfront_origin_request_policy.origin_request_policy.id
```

Comment:
These two arguments need to be moved within the `default_cache_behavior` block of the `aws_cloudfront_distribution` resource or the configuration isn't valid:

```suggestion
```